README.md

```diff
@@ -45,7 +45,12 @@ This will let you gauge the impact of a change before actually touching
 any of your nodes.
 
 This tool is delivered as a collection of Puppet Faces.
-It thus requires a Puppet installation to properly run.
+It thus requires a Puppet 6.11 (or newer) installation to properly run.
+
+Only the system that runs the catalog-diff needs to be Puppet 6.11 or newer.
+You can run it on a Puppetserver that also provides you the old or new
+catalogs. However, you can also get catalogs from remote
+Puppetservers/PuppetDBs. PuppetDB 2.3 and newer are supported.
 
```

Comment:
Im not sure what you are trying to say here.  perhaps something like?

>Only the system that runs catalog-diff needs to be using Puppet 6.11 or newer.
>The puppetservers you are targeting to do the catalog compilation can be running 
>an early version of puppet; however puppetdb must be at least version 2.3

Reply:
This diff was coming from another PR. I updated it here: https://github.com/voxpupuli/puppet-catalog-diff/pull/71